schema/geometry_schema.json

```diff
@@ -623,6 +635,45 @@
 				"id",
 				"name"
 			]
+		},
+		"PitchedRoofType": {
+			"properties": {
+				"id": {
+					"type": "string"
+				},
+				"name": {
+					"type": [
+						"string",
+						"null"
+					]
+				},
+				"type": {
```

Comment:
@shorowit would it be possible to name this something else? We're already using `type` internally to as a kind of tag discriminator. Eg: `if (obj.type === 'space') { ... }`